### PR TITLE
fix package name in ci workflow

### PR DESCRIPTION
**Pull request by flynneva**
Signed-off-by: Evan Flynn <evan.flynn@apex.ai>

### CODE REVIEW

**Comment by flynneva** [comment]
@LanderU the ci should now run for all active ROS2 versions, apologies for all the pings/notifications 😓  its erroring now because we dont have any code in this repo until we merge over from `ApexAI/reference-system-autoware`

**Comment by flynneva** [comment]
for reference im just going off what I setup for the [`usb_cam` package](https://github.com/ros-drivers/usb_cam/tree/ros2)

File: .github/workflows/colcon-build.yml
Comment (LanderU) [inline .github/workflows/colcon-build.yml:44]:
```suggestion
        run: mkdir -p ros2_ws/src
```

File: .github/workflows/colcon-build.yml
Comment (LanderU) [inline .github/workflows/colcon-build.yml:48]:
```suggestion
          path: ros2_ws/src
```

**Comment by LanderU** [comment]
@flynneva, no problem with the pings :), do you have an idea when do you include a package here? We can merge this and then adjust when the package/s comes.

**Comment by flynneva** [comment]
I had the same question - maybe after the two open PRs get merged in then we can bring over the repo from the ApexAI fork

**Comment by flynneva** [comment]
@LanderU do you think you could merge this in so we could test #9 properly?

File: .github/workflows/colcon-build.yml
Comment (LanderU) [inline .github/workflows/colcon-build.yml:52]:
```suggestion
          package-name: reference_system
```

**Comment by LanderU** [comment]
@flynneva, if you apply the change I'll merge this.

**Review by LanderU — APPROVED** [review]
LGTM!

**Pull request merged**
